Evergreen/Manifests/PSFPython.json

```diff
@@ -0,0 +1,25 @@
+{
+	"Name": "PSF Python",
+	"Source": "https://www.python.org/",
+	"Get": {
```

Comment:
This should look similar to the following (I need to update documentation on the recommended manifest structure):
```json
"Get": {
    "Update": {
        "Uri": "https://www.python.org/api/v2/downloads/release/",
        "MatchRelease" : "(?<=\\/downloads\\/release\\/)(\\d+)"
    },
    "Download": {
        "Uri": "https://www.python.org/api/v2/downloads/release_file/",
        "DatePattern": "yyyy-MM-ddTHH:mm:ssK",
        "MatchFileTypes": "\\.exe$|\\.msi$",
        "MatchVersion": "(\\d+(\\.\\d+){1,3})"
    },
}
```